symfony/console/3.3/bin/console

```diff
@@ -33,6 +33,6 @@ if ($_SERVER['APP_DEBUG']) {
     }
 }
 
-$kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+$kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG'], dirname(__DIR__));
```

Comment:
shouldn't it be part of a 4.3+ recipe only instead ?

Reply:
Same for the other entrypoint then (`public/index.php`) I guess.

> To me: 3.3

That's been done acting on @nicolas-grekas's recommendation above. I'll let him give his own rationale 👍

Reply:
Duplicating all the files in the recipe is not worth it.